index.d.ts

```diff
@@ -17,6 +17,8 @@ interface TypeOption {
 
 	The `distinguishable` set contains only uppercase characters that are not easily confused: `CDEHKMPRTUWXY012458`. It can be useful if you need to print out a short string that you'd like users to read and type back in with minimal errors. For example, reading a code off of a screen that needs to be typed into a phone to connect two devices.
 
+	The `ascii-printable` set contains all [printable ascii characters](https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters): `` !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~`` It can be useful for generating passwords where all possible ASCII characters should be used.
```

Comment:
```suggestion
	The `ascii-printable` set contains all [printable ASCII characters](https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters): `` !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~`` It can be useful for generating passwords where all possible ASCII characters should be used.
```